[2023-12-25T10:19:21.649+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL.Fetch_from_Postgresql manual__2023-12-25T10:19:20.438953+00:00 [queued]>
[2023-12-25T10:19:21.656+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: ETL.Fetch_from_Postgresql manual__2023-12-25T10:19:20.438953+00:00 [queued]>
[2023-12-25T10:19:21.656+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-12-25T10:19:21.656+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2023-12-25T10:19:21.656+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-12-25T10:19:21.663+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): Fetch_from_Postgresql> on 2023-12-25 10:19:20.438953+00:00
[2023-12-25T10:19:21.669+0000] {standard_task_runner.py:52} INFO - Started process 808 to run task
[2023-12-25T10:19:21.672+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'Fetch_from_Postgresql', 'manual__2023-12-25T10:19:20.438953+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_saepul_hilal_DAG.py', '--cfg-path', '/tmp/tmpvvl_3mbi', '--error-file', '/tmp/tmpl4h1des0']
[2023-12-25T10:19:21.674+0000] {standard_task_runner.py:80} INFO - Job 7: Subtask Fetch_from_Postgresql
[2023-12-25T10:19:21.713+0000] {task_command.py:371} INFO - Running <TaskInstance: ETL.Fetch_from_Postgresql manual__2023-12-25T10:19:20.438953+00:00 [running]> on host 0df44d59871b
[2023-12-25T10:19:21.761+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=saepul
AIRFLOW_CTX_DAG_ID=ETL
AIRFLOW_CTX_TASK_ID=Fetch_from_Postgresql
AIRFLOW_CTX_EXECUTION_DATE=2023-12-25T10:19:20.438953+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-25T10:19:20.438953+00:00
[2023-12-25T10:19:21.866+0000] {python.py:173} INFO - Done. Returned value was: None
[2023-12-25T10:19:21.873+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=Fetch_from_Postgresql, execution_date=20231225T101920, start_date=20231225T101921, end_date=20231225T101921
[2023-12-25T10:19:21.890+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-25T10:19:21.913+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
